opentracing/src/main/java/io/opentracing/noop/NoopSpan.java

```diff
@@ -60,4 +63,9 @@ public Span log(String eventName, Object payload) {
     public Span log(long timestampMicroseconds, String eventName, Object payload) {
         return this;
     }
+
+	@Override
+	public Set<String> getBaggageKeys() {
+		return Collections.emptySet();
+	}
```

Comment:
Do you have an example of when and why this would be used?
I'm unsure of a situation where you'd need to *explore* what baggage existed.